docs/meshcloud.payment-methods.md

```diff
@@ -0,0 +1,94 @@
+---
+id: meshstack.payment-methods
+title: Payment Methods
+---
+
+## Introduction
+
+Just like you pay for a new t-shirt online via your credit card, enterprises also have to be responsible for the costs that they make in cloud platforms. Payment methods are used for allocating cloud costs, all the way back to the responsible department of your organization, for a clear and transparent accounting process. As the cloud can be mighty with its enormous offering of practically unlimited services, it is important that all meshProjects and meshCustomers are correctly charged and that this information makes its way back to the required systems. Payment methods therefore are the link between the cloud usage and the resulting cost in the context of a meshProject.
+
+Payment methods are created **per** meshCustomer and come in two variants:
+
+1) A cost center payment method (indicated via `COST_CENTER`), meaning the costs made by the meshTenants in your meshProjects will be fully charged to the cost center of your organization.
+
+2) A budget payment method (indicated via `COST_LIMITATION`), meaning costs made by the meshTenants in your meshProjects will be charged based on the monetary budget amount that is set on this payment method.
+
+![Payment Methods Lifecycle](assets/payment_methods/payment_method_overall_lifecycle.png)
+
+## The Payment Method Lifecycle: Creation
```

Comment:
d: Why is it called `lifecylce`? I understand those parts of the lifecycle more as use cases. Also, lifecycle for me means that there should be info on when it's destroyed / deleted.

Reply:
Well you can set an expiration date ;-) I like "lifecylce" because it emphasizes the different states of a landing zone from partner to customer to project. In the end "How it comes to life". We should also have a delete or disable payment methods in the future. I agree on that

Reply:
I also prefer lifecyle. As we will probably introduce a deletion/disabling of payment methods in the future I think it makes sense.